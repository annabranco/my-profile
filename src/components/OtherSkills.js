import React from 'react';
import '../styles/components/OtherSkills.css';
import MacBar from '../images/bg/bg-mac.jpg';

class OtherSkills extends React.Component {

	constructor(props) {
		super(props)

		this.state = {
			opened: false,
			read: false
		}
	}


		open = () => {
			if (!this.state.opened) {

				this.setState({
					opened: true,
					read: true
				 });
				document.querySelector('.section__otherSkills').classList.add('opened');
				document.querySelector('.otherSkills__mac').classList.add('opened');
				document.querySelector('.otherSkills__mac--screen').classList.add('opened');
				document.querySelector('.otherSkills__mac--keyboard').classList.add('hidden');
				document.querySelector('.otherSkills__mac--keyboard-keyArea').classList.add('hidden');
				this.props.markAsRead();

			} else {

				this.setState({ opened: false });
				document.querySelector('.section__otherSkills').classList.remove('opened');
				document.querySelector('.otherSkills__mac').classList.remove('opened');
				document.querySelector('.otherSkills__mac--screen').classList.remove('opened');
				document.querySelector('.otherSkills__mac--keyboard').classList.remove('hidden');
				document.querySelector('.otherSkills__mac--keyboard-keyArea').classList.remove('hidden');
			}
		}

	render () {

		const text = this.props.texts.OtherSkills;
		const language = this.props.language;

		return (

			<React.Fragment>

				{!this.props.viewedOtherSkills && !this.state.read ?
					<React.Fragment>
						<p className="seabed__findSomething">{text[language].find}</p>
						<p className="seabed__findSomething">{text[language].find2}</p>
						<p className="seabed__findSomething">{text[language].investigate}</p>
					</React.Fragment>
				: null
				}

				<div className="otherSkills__mac">
					<div className="otherSkills__mac--screen">

						<section className="section__otherSkills" onClick={this.open}>

							<img src={MacBar} alt="" className="skills--topBar"/>
							<p className="seabed__click2close seabed__click2close-otherSkills">{this.props.texts.Seabed[language].click2close}</p>

							<div className="skills__outer skills__outer--languages">
								<h2 className="skills__table--title">{text[language].languages}</h2>
								<table className="skills__table skills__table--languages">
									<tbody>
										<tr>
										    <td>{text[language].english}<img src="https://www.countryflags.io/us/flat/16.png" alt="" className="languages--flag" /></td>
										    <td><i className="fas fa-comments icon--languages" title="Speaking"></i>{text[language].fluent}</td>
										    <td><i className="fas fa-book-open icon--languages" title="Reading"></i>{text[language].fluent}</td>
											<td><i className="fas fa-pen icon--languages" title="Writing"></i>{text[language].fluent}</td>
										</tr>
										<tr>
										    <td>{text[language].spanish}<img src="https://www.countryflags.io/es/flat/16.png" alt="" className="languages--flag" /></td>
										    <td><i className="fas fa-comments icon--languages" title="Speaking"></i>{text[language].fluent}</td>
										    <td><i className="fas fa-book-open icon--languages" title="Reading"></i>{text[language].fluent}</td>
											<td><i className="fas fa-pen icon--languages" title="Writing"></i>{text[language].fluent}</td>
										</tr>
										<tr>
											<td>{text[language].portuguese}<img src="https://www.countryflags.io/br/flat/16.png" alt="" className="languages--flag" /></td>
											<td><i className="fas fa-comments icon--languages" title="Speaking"></i>{text[language].fluent}</td>
											<td><i className="fas fa-book-open icon--languages" title="Reading"></i>{text[language].fluent}</td>
											<td><i className="fas fa-pen icon--languages" title="Writing"></i>{text[language].fluent}</td>
										</tr>
										<tr>
											<td>{text[language].french}<img src="https://www.countryflags.io/fr/flat/16.png" alt="" className="languages--flag" /></td>
											<td><i className="fas fa-comments icon--languages" title="Speaking"></i>{text[language].intermediate}</td>
											<td><i className="fas fa-book-open icon--languages" title="Reading"></i>{text[language].intermediate}</td>
											<td><i className="fas fa-pen icon--languages" title="Writing"></i>{text[language].basic}</td>
										</tr>
									 	<tr>
											<td>{text[language].russian}<img src="https://www.countryflags.io/ru/flat/16.png" alt="" className="languages--flag" /></td>
											<td><i className="fas fa-comments icon--languages" title="Speaking"></i>{text[language].basic}</td>
											<td><i className="fas fa-book-open icon--languages" title="Reading"></i>{text[language].basic}</td>
											<td><i className="fas fa-pen icon--languages" title="Writing"></i>{text[language].basic}</td>
									 	</tr>
									</tbody>
								</table>
							</div>

							<div className="skills__outer skills__outer--other">
								<h2 className="skills__table--title">{text[language].other}</h2>
								<table className="skills__table skills__table--other">
									<thead>
										<tr>
											<th>{text[language].skill}</th>
											<th>{text[language].how}</th>
										</tr>
									</thead>
									<tbody>
										<tr>
											<td>{text[language].skill1}</td>
											<td>{text[language].skill1details}</td>
										</tr>
										<tr>
											<td>{text[language].skill2}</td>
											<td>{text[language].skill2details}</td>
										</tr>
										<tr>
											<td>{text[language].skill3}</td>
											<td>{text[language].skill3details}</td>
										</tr>
										<tr>
											<td>{text[language].skill4}</td>
											<td>{text[language].skill4details}</td>
										</tr>
										<tr>
											<td>{text[language].skill5}</td>
											<td>{text[language].skill5details}</td>
										</tr>
									</tbody>
								</table>
							</div>
							<div className="skills__outer skills__outer--design">
								<h2 className="skills__table--title">{text[language].design}</h2>
								<p className="skills__design">{text[language].samples}</p>
								<div className="skills__design--samples">
								<img src="https://via.placeholder.com/300x200/fff5f4/000?text=sample" alt="" className="design__sample"/>
								<img src="https://via.placeholder.com/300x200/fff5f4/000?text=sample" alt="" className="design__sample"/>
								<img src="https://via.placeholder.com/300x200/fff5f4/000?text=sample" alt="" className="design__sample"/>
								<img src="https://via.placeholder.com/300x200/fff5f4/000?text=sample" alt="" className="design__sample"/>
								<img src="https://via.placeholder.com/300x200/fff5f4/000?text=sample" alt="" className="design__sample"/>
								<img src="https://via.placeholder.com/300x200/fff5f4/000?text=sample" alt="" className="design__sample"/>
								</div>
							</div>


						</section>
					</div>
					<div className="otherSkills__mac--keyboard"></div>
					<div className="otherSkills__mac--keyboard-keyArea">
						<p className="otherSkills__mac--keyboard-keys"> . . . . . . . ...</p>
						<p className="otherSkills__mac--keyboard-keys">... . . . . . . ..</p>
						<p className="otherSkills__mac--keyboard-keys">. . . . . . . . . .</p>
						<p className="otherSkills__mac--keyboard-keys">. . . . . . . . . .</p>
						<p className="otherSkills__mac--keyboard-keys">--------- ......</p>
					</div>
				</div>

			</React.Fragment>

		);
	}
}

export default OtherSkills;
