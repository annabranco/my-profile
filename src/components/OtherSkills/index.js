import React from 'react';
import { useSelector } from 'react-redux';
import { func } from 'prop-types';
import {
  globalTextsSelector,
  otherSkillsTextSelector
} from '../../redux/selectors';
import { isDesktop } from '../../utils/device';
import { MacNotebook } from '../../assets/images';
import { TextFindSomething } from '../Seabed/styles';
import {
  CloseButton,
  DesignArea,
  DesignText,
  Flag,
  Icon,
  KeyboardKeysWrapper,
  Keys,
  LanguagesArea,
  LanguagesTable,
  MacKeyboard,
  MacScreen,
  OtherInformationsArea,
  OtherInformationsTable,
  OtherSkillsWrapper,
  Sample,
  SamplesWrapper,
  SectionOtherSkills,
  Title,
  TopBar
} from './styles';
import { seabedElementsPropType } from '../../types';

const OtherSkills = ({
  onClickClose,
  onClickOpen,
  status: { read, visible }
}) => {
  const texts = useSelector(otherSkillsTextSelector);
  const globalTexts = useSelector(globalTextsSelector);

  return (
    <>
      {!read && isDesktop ? (
        <>
          <TextFindSomething>{texts.find}</TextFindSomething>
          <TextFindSomething>{texts.find2}</TextFindSomething>
          <TextFindSomething>{texts.investigate}</TextFindSomething>
        </>
      ) : null}

      <SectionOtherSkills id="Other Skills Section" visible={visible}>
        {isDesktop && <TopBar visible={visible} src={MacNotebook} alt="" />}
        <MacScreen visible={visible}>
          <OtherSkillsWrapper
            aria-label={globalTexts.open}
            onClick={() => onClickOpen('otherSkillsSection')}
            role="button"
            tabIndex={0}
            visible={visible}
          >
            <LanguagesArea>
              <Title>{texts.languages}</Title>
              <LanguagesTable>
                <tbody>
                  <tr>
                    <td>
                      {texts.english}
                      <Flag
                        src="https://www.countryflags.io/us/flat/16.png"
                        alt=""
                      />
                    </td>
                    <td>
                      <Icon className="fas fa-comments" title="Speaking" />
                      {texts.fluent}
                    </td>
                    <td>
                      <Icon className="fas fa-book-open" title="Reading" />
                      {texts.fluent}
                    </td>
                    <td>
                      <Icon className="fas fa-pen" title="Writing" />
                      {texts.fluent}
                    </td>
                  </tr>
                  <tr>
                    <td>
                      {texts.spanish}
                      <Flag
                        src="https://www.countryflags.io/es/flat/16.png"
                        alt=""
                      />
                    </td>
                    <td>
                      <Icon className="fas fa-comments" title="Speaking" />
                      {texts.fluent}
                    </td>
                    <td>
                      <Icon className="fas fa-book-open" title="Reading" />
                      {texts.fluent}
                    </td>
                    <td>
                      <Icon className="fas fa-pen" title="Writing" />
                      {texts.fluent}
                    </td>
                  </tr>
                  <tr>
                    <td>
                      {texts.portuguese}
                      <Flag
                        src="https://www.countryflags.io/br/flat/16.png"
                        alt=""
                      />
                    </td>
                    <td>
                      <Icon className="fas fa-comments" title="Speaking" />
                      {texts.fluent}
                    </td>
                    <td>
                      <Icon className="fas fa-book-open" title="Reading" />
                      {texts.fluent}
                    </td>
                    <td>
                      <Icon className="fas fa-pen" title="Writing" />
                      {texts.fluent}
                    </td>
                  </tr>
                  <tr>
                    <td>
                      {texts.french}
                      <Flag
                        src="https://www.countryflags.io/fr/flat/16.png"
                        alt=""
                      />
                    </td>
                    <td>
                      <Icon className="fas fa-comments" title="Speaking" />
                      {texts.intermediate}
                    </td>
                    <td>
                      <Icon className="fas fa-book-open" title="Reading" />
                      {texts.intermediate}
                    </td>
                    <td>
                      <Icon className="fas fa-pen" title="Writing" />
                      {texts.basic}
                    </td>
                  </tr>
                  <tr>
                    <td>
                      {texts.russian}
                      <Flag
                        src="https://www.countryflags.io/ru/flat/16.png"
                        alt=""
                      />
                    </td>
                    <td>
                      <Icon className="fas fa-comments" title="Speaking" />
                      {texts.basic}
                    </td>
                    <td>
                      <Icon className="fas fa-book-open" title="Reading" />
                      {texts.basic}
                    </td>
                    <td>
                      <Icon className="fas fa-pen" title="Writing" />
                      {texts.basic}
                    </td>
                  </tr>
                </tbody>
              </LanguagesTable>
            </LanguagesArea>

            <OtherInformationsArea>
              <Title>{texts.other}</Title>
              <OtherInformationsTable>
                <thead>
                  <tr>
                    <th>{texts.skill}</th>
                    <th>{texts.how}</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>{texts.skill1}</td>
                    <td>{texts.skill1details}</td>
                  </tr>
                  <tr>
                    <td>{texts.skill2}</td>
                    <td>{texts.skill2details}</td>
                  </tr>
                  <tr>
                    <td>{texts.skill3}</td>
                    <td>{texts.skill3details}</td>
                  </tr>
                  <tr>
                    <td>{texts.skill4}</td>
                    <td>{texts.skill4details}</td>
                  </tr>
                  <tr>
                    <td>{texts.skill5}</td>
                    <td>{texts.skill5details}</td>
                  </tr>
                </tbody>
              </OtherInformationsTable>
            </OtherInformationsArea>
            <DesignArea>
              <Title>{texts.design}</Title>
              <DesignText>{texts.samples}</DesignText>
              <SamplesWrapper>
                <a
                  href="https://pre00.deviantart.net/f5fe/th/pre/f/2018/065/a/2/rio_de_janeiro__um_retrato_da_violencia_by_annabranco-dc53j1i.jpg"
                  rel="noopener noreferrer"
                  target="_Blank"
                >
                  <Sample image="https://pre00.deviantart.net/f5fe/th/pre/f/2018/065/a/2/rio_de_janeiro__um_retrato_da_violencia_by_annabranco-dc53j1i.jpg" />
                </a>
                <a
                  href="https://img00.deviantart.net/9fb7/i/2018/069/2/c/recuerda_siempre_mirar_el_color_de_las_banderas_by_annabranco-dc5gv8e.jpg"
                  rel="noopener noreferrer"
                  target="_Blank"
                >
                  <Sample image="https://img00.deviantart.net/9fb7/i/2018/069/2/c/recuerda_siempre_mirar_el_color_de_las_banderas_by_annabranco-dc5gv8e.jpg" />
                </a>
                <a
                  href="https://pre00.deviantart.net/3afa/th/pre/i/2018/069/e/3/basic_english_by_annabranco-dc5go6q.jpg"
                  rel="noopener noreferrer"
                  target="_Blank"
                >
                  <Sample image="https://pre00.deviantart.net/3afa/th/pre/i/2018/069/e/3/basic_english_by_annabranco-dc5go6q.jpg" />
                </a>
                <a
                  href="https://pre00.deviantart.net/6e8f/th/pre/f/2018/245/e/8/violencia1_en_by_annabranco-dclta8q.jpg"
                  rel="noopener noreferrer"
                  target="_Blank"
                >
                  <Sample image="https://pre00.deviantart.net/6e8f/th/pre/f/2018/245/e/8/violencia1_en_by_annabranco-dclta8q.jpg" />
                </a>
                <a
                  href="https://pre00.deviantart.net/9f40/th/pre/f/2018/065/1/7/terremoto_en_mexico_19_09_17_by_alvarobranco_dbo6p_by_annabranco-dc53k8f.jpg"
                  rel="noopener noreferrer"
                  target="_Blank"
                >
                  <Sample image="https://pre00.deviantart.net/9f40/th/pre/f/2018/065/1/7/terremoto_en_mexico_19_09_17_by_alvarobranco_dbo6p_by_annabranco-dc53k8f.jpg" />
                </a>
                <a
                  href="https://pre00.deviantart.net/d51e/th/pre/i/2018/069/5/b/european_map__arabic__by_annabranco-dc5gve5.jpg"
                  rel="noopener noreferrer"
                  target="_Blank"
                >
                  <Sample image="https://pre00.deviantart.net/d51e/th/pre/i/2018/069/5/b/european_map__arabic__by_annabranco-dc5gve5.jpg" />
                </a>
              </SamplesWrapper>
            </DesignArea>
            <CloseButton
              aria-label={globalTexts.close}
              hidden={!visible}
              onClick={() => onClickClose('otherSkillsSection')}
              type="button"
            >
              {globalTexts.close}
            </CloseButton>
          </OtherSkillsWrapper>
        </MacScreen>
        <MacKeyboard hidden={visible} />
        <KeyboardKeysWrapper hidden={visible}>
          <Keys> . . . . . . . ...</Keys>
          <Keys>... . . . . . . ..</Keys>
          <Keys>. . . . . . . . . .</Keys>
          <Keys>. . . . . . . . . .</Keys>
          <Keys>------......</Keys>
        </KeyboardKeysWrapper>
      </SectionOtherSkills>
    </>
  );
};

OtherSkills.propTypes = {
  onClickClose: func.isRequired,
  onClickOpen: func.isRequired,
  status: seabedElementsPropType.isRequired
};

export default OtherSkills;
