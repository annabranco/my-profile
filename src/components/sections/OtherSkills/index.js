import React from 'react';
import { useSelector } from 'react-redux';
import { func } from 'prop-types';
import { otherSkillsTextSelector } from '../../../redux/selectors';
import { isDesktop } from '../../../utils/device';
import OtherSkillsInfo from './OtherSkillsInfo';
import { MacNotebook } from '../../../assets/images';
import { seabedElementsPropType } from '../../../types/propTypes';
import { TextFindSomething } from '../Seabed/styles';
import {
  KeyboardKeysWrapper,
  Keys,
  MacKeyboard,
  MacScreen,
  SectionOtherSkills,
  TopBar
} from './styles';

const OtherSkills = ({
  onClickClose,
  onClickOpen,
  status: { read, visible }
}) => {
  const texts = useSelector(otherSkillsTextSelector);

  return (
    <>
      {!read && isDesktop ? (
        <>
          <TextFindSomething>{texts.find}</TextFindSomething>
          <TextFindSomething>{texts.find2}</TextFindSomething>
          <TextFindSomething>{texts.investigate}</TextFindSomething>
        </>
      ) : null}

      <SectionOtherSkills id="Other Skills Section" visible={visible}>
        {isDesktop && <TopBar visible={visible} src={MacNotebook} alt="" />}
        <MacScreen visible={visible}>
          <OtherSkillsInfo
            onClickOpen={onClickOpen}
            onClickClose={onClickClose}
            visible={visible}
          />
        </MacScreen>
        <MacKeyboard hidden={visible} />
        <KeyboardKeysWrapper hidden={visible}>
          <Keys> . . . . . . . ...</Keys>
          <Keys>... . . . . . . ..</Keys>
          <Keys>. . . . . . . . . .</Keys>
          <Keys>. . . . . . . . . .</Keys>
          <Keys>------......</Keys>
        </KeyboardKeysWrapper>
      </SectionOtherSkills>
    </>
  );
};

OtherSkills.propTypes = {
  onClickClose: func.isRequired,
  onClickOpen: func.isRequired,
  status: seabedElementsPropType.isRequired
};

export default OtherSkills;
