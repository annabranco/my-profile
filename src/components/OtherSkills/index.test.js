import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';
import '../../../testing/mocks/matchMedia';
import OtherSkills from '.';
import { setupMockProvider, subSectionsStatus } from '../../../testing/mocks';

// import { getMockState } from '../../../testing/mocks/mockState';

describe('< OtherSkills >', () => {
  let MockProvider;
  // let MOCK_STATE;
  let component;

  beforeAll(() => {
    MockProvider = setupMockProvider({ language: 'en' }).MockProvider;
    // MOCK_STATE = getMockState({ language: 'en' });

    component = mount(
      <MockProvider>
        <OtherSkills
          onClickClose={() => null}
          onClickOpen={() => null}
          status={subSectionsStatus}
        />
      </MockProvider>
    );
  });

  it('should mount', () => {
    expect(component).toBeDefined();
  });

  it('matches the snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
