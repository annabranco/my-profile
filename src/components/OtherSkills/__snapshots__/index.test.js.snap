// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< OtherSkills > matches the snapshot 1`] = `
<MockProvider>
  <Provider
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  >
    <OtherSkills
      onClickClose={[Function]}
      onClickOpen={[Function]}
      status={
        Object {
          "active": false,
          "read": false,
          "visible": false,
        }
      }
    >
      <Text Find Something>
        <p
          className="gcXdCD"
        >
          You've found... a computer?!
        </p>
      </Text Find Something>
      <Text Find Something>
        <p
          className="gcXdCD"
        >
          And it is oddly on
        </p>
      </Text Find Something>
      <Text Find Something>
        <p
          className="gcXdCD"
        >
          Click on the screen to investigate it.
        </p>
      </Text Find Something>
      <OTHER SKILLS Section
        id="Other Skills Section"
        visible={false}
      >
        <section
          className="jkdJcP"
          id="Other Skills Section"
        >
          <Top Bar
            alt=""
            src={
              Object {
                "process": [Function],
              }
            }
            visible={false}
          >
            <img
              alt=""
              className="RNSqc"
              src={
                Object {
                  "process": [Function],
                }
              }
            />
          </Top Bar>
          <Mac Screen
            visible={false}
          >
            <div
              className="fTSRkD"
            >
              <Other Skills Wrapper
                aria-label="Open"
                onClick={[Function]}
                role="button"
                tabIndex={0}
                visible={false}
              >
                <div
                  aria-label="Open"
                  className="MmBJI"
                  onClick={[Function]}
                  role="button"
                  tabIndex={0}
                >
                  <Languages Area>
                    <div
                      className="dWUhcF euwOso"
                    >
                      <Title>
                        <h2
                          className="fgVbma"
                        >
                          Languages
                        </h2>
                      </Title>
                      <Languages Table>
                        <table
                          className="hTOmvu emKENg"
                        >
                          <tbody>
                            <tr>
                              <td>
                                English
                                <Flag
                                  alt=""
                                  src="https://www.countryflags.io/us/flat/16.png"
                                >
                                  <img
                                    alt=""
                                    className="ibQKQf"
                                    src="https://www.countryflags.io/us/flat/16.png"
                                  />
                                </Flag>
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-comments"
                                  title="Speaking"
                                >
                                  <i
                                    className="idlvvH fas fa-comments"
                                    title="Speaking"
                                  />
                                </Icon>
                                Fluent
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-book-open"
                                  title="Reading"
                                >
                                  <i
                                    className="idlvvH fas fa-book-open"
                                    title="Reading"
                                  />
                                </Icon>
                                Fluent
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-pen"
                                  title="Writing"
                                >
                                  <i
                                    className="idlvvH fas fa-pen"
                                    title="Writing"
                                  />
                                </Icon>
                                Fluent
                              </td>
                            </tr>
                            <tr>
                              <td>
                                Spanish
                                <Flag
                                  alt=""
                                  src="https://www.countryflags.io/es/flat/16.png"
                                >
                                  <img
                                    alt=""
                                    className="ibQKQf"
                                    src="https://www.countryflags.io/es/flat/16.png"
                                  />
                                </Flag>
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-comments"
                                  title="Speaking"
                                >
                                  <i
                                    className="idlvvH fas fa-comments"
                                    title="Speaking"
                                  />
                                </Icon>
                                Fluent
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-book-open"
                                  title="Reading"
                                >
                                  <i
                                    className="idlvvH fas fa-book-open"
                                    title="Reading"
                                  />
                                </Icon>
                                Fluent
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-pen"
                                  title="Writing"
                                >
                                  <i
                                    className="idlvvH fas fa-pen"
                                    title="Writing"
                                  />
                                </Icon>
                                Fluent
                              </td>
                            </tr>
                            <tr>
                              <td>
                                Portuguese
                                <Flag
                                  alt=""
                                  src="https://www.countryflags.io/br/flat/16.png"
                                >
                                  <img
                                    alt=""
                                    className="ibQKQf"
                                    src="https://www.countryflags.io/br/flat/16.png"
                                  />
                                </Flag>
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-comments"
                                  title="Speaking"
                                >
                                  <i
                                    className="idlvvH fas fa-comments"
                                    title="Speaking"
                                  />
                                </Icon>
                                Fluent
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-book-open"
                                  title="Reading"
                                >
                                  <i
                                    className="idlvvH fas fa-book-open"
                                    title="Reading"
                                  />
                                </Icon>
                                Fluent
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-pen"
                                  title="Writing"
                                >
                                  <i
                                    className="idlvvH fas fa-pen"
                                    title="Writing"
                                  />
                                </Icon>
                                Fluent
                              </td>
                            </tr>
                            <tr>
                              <td>
                                French
                                <Flag
                                  alt=""
                                  src="https://www.countryflags.io/fr/flat/16.png"
                                >
                                  <img
                                    alt=""
                                    className="ibQKQf"
                                    src="https://www.countryflags.io/fr/flat/16.png"
                                  />
                                </Flag>
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-comments"
                                  title="Speaking"
                                >
                                  <i
                                    className="idlvvH fas fa-comments"
                                    title="Speaking"
                                  />
                                </Icon>
                                Intermediate
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-book-open"
                                  title="Reading"
                                >
                                  <i
                                    className="idlvvH fas fa-book-open"
                                    title="Reading"
                                  />
                                </Icon>
                                Intermediate
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-pen"
                                  title="Writing"
                                >
                                  <i
                                    className="idlvvH fas fa-pen"
                                    title="Writing"
                                  />
                                </Icon>
                                Basic
                              </td>
                            </tr>
                            <tr>
                              <td>
                                Russian
                                <Flag
                                  alt=""
                                  src="https://www.countryflags.io/ru/flat/16.png"
                                >
                                  <img
                                    alt=""
                                    className="ibQKQf"
                                    src="https://www.countryflags.io/ru/flat/16.png"
                                  />
                                </Flag>
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-comments"
                                  title="Speaking"
                                >
                                  <i
                                    className="idlvvH fas fa-comments"
                                    title="Speaking"
                                  />
                                </Icon>
                                Basic
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-book-open"
                                  title="Reading"
                                >
                                  <i
                                    className="idlvvH fas fa-book-open"
                                    title="Reading"
                                  />
                                </Icon>
                                Basic
                              </td>
                              <td>
                                <Icon
                                  className="fas fa-pen"
                                  title="Writing"
                                >
                                  <i
                                    className="idlvvH fas fa-pen"
                                    title="Writing"
                                  />
                                </Icon>
                                Basic
                              </td>
                            </tr>
                          </tbody>
                        </table>
                      </Languages Table>
                    </div>
                  </Languages Area>
                  <Other Informations Area>
                    <div
                      className="dWUhcF cugJmd"
                    >
                      <Title>
                        <h2
                          className="fgVbma"
                        >
                          Other skills
                        </h2>
                      </Title>
                      <Other Informations Table>
                        <table
                          className="hTOmvu jciiQn"
                        >
                          <thead>
                            <tr>
                              <th>
                                Skill
                              </th>
                              <th>
                                How I have developed it
                              </th>
                            </tr>
                          </thead>
                          <tbody>
                            <tr>
                              <td>
                                Working under pressure
                              </td>
                              <td>
                                More than 5 years as volunteer on ambulances and emergencies
                              </td>
                            </tr>
                            <tr>
                              <td>
                                Leadership
                              </td>
                              <td>
                                More than 3 years of professional experience managing different teams
                              </td>
                            </tr>
                            <tr>
                              <td>
                                Communication
                              </td>
                              <td>
                                Solid experience teaching, giving formations and speeches
                              </td>
                            </tr>
                            <tr>
                              <td>
                                Empathy
                              </td>
                              <td>
                                Psychology formation and humanitarian background
                              </td>
                            </tr>
                            <tr>
                              <td>
                                Flexibility
                              </td>
                              <td>
                                Very good experience working within multicultural environments
                              </td>
                            </tr>
                          </tbody>
                        </table>
                      </Other Informations Table>
                    </div>
                  </Other Informations Area>
                  <Design Area>
                    <div
                      className="dWUhcF gdJpcf"
                    >
                      <Title>
                        <h2
                          className="fgVbma"
                        >
                          Graphic design
                        </h2>
                      </Title>
                      <Design Text>
                        <p
                          className="hnjRhx"
                        >
                          Here are some of my design creations:
                        </p>
                      </Design Text>
                      <Samples Wrapper>
                        <div
                          className="kOXSrB"
                        >
                          <a
                            href="https://pre00.deviantart.net/f5fe/th/pre/f/2018/065/a/2/rio_de_janeiro__um_retrato_da_violencia_by_annabranco-dc53j1i.jpg"
                            rel="noopener noreferrer"
                            target="_Blank"
                          >
                            <Sample
                              image="https://pre00.deviantart.net/f5fe/th/pre/f/2018/065/a/2/rio_de_janeiro__um_retrato_da_violencia_by_annabranco-dc53j1i.jpg"
                            >
                              <div
                                className="bUGLvw"
                              />
                            </Sample>
                          </a>
                          <a
                            href="https://img00.deviantart.net/9fb7/i/2018/069/2/c/recuerda_siempre_mirar_el_color_de_las_banderas_by_annabranco-dc5gv8e.jpg"
                            rel="noopener noreferrer"
                            target="_Blank"
                          >
                            <Sample
                              image="https://img00.deviantart.net/9fb7/i/2018/069/2/c/recuerda_siempre_mirar_el_color_de_las_banderas_by_annabranco-dc5gv8e.jpg"
                            >
                              <div
                                className="ehxKDO"
                              />
                            </Sample>
                          </a>
                          <a
                            href="https://pre00.deviantart.net/3afa/th/pre/i/2018/069/e/3/basic_english_by_annabranco-dc5go6q.jpg"
                            rel="noopener noreferrer"
                            target="_Blank"
                          >
                            <Sample
                              image="https://pre00.deviantart.net/3afa/th/pre/i/2018/069/e/3/basic_english_by_annabranco-dc5go6q.jpg"
                            >
                              <div
                                className="hPiMEm"
                              />
                            </Sample>
                          </a>
                          <a
                            href="https://pre00.deviantart.net/6e8f/th/pre/f/2018/245/e/8/violencia1_en_by_annabranco-dclta8q.jpg"
                            rel="noopener noreferrer"
                            target="_Blank"
                          >
                            <Sample
                              image="https://pre00.deviantart.net/6e8f/th/pre/f/2018/245/e/8/violencia1_en_by_annabranco-dclta8q.jpg"
                            >
                              <div
                                className="evwDjm"
                              />
                            </Sample>
                          </a>
                          <a
                            href="https://pre00.deviantart.net/9f40/th/pre/f/2018/065/1/7/terremoto_en_mexico_19_09_17_by_alvarobranco_dbo6p_by_annabranco-dc53k8f.jpg"
                            rel="noopener noreferrer"
                            target="_Blank"
                          >
                            <Sample
                              image="https://pre00.deviantart.net/9f40/th/pre/f/2018/065/1/7/terremoto_en_mexico_19_09_17_by_alvarobranco_dbo6p_by_annabranco-dc53k8f.jpg"
                            >
                              <div
                                className="gnSGtx"
                              />
                            </Sample>
                          </a>
                          <a
                            href="https://pre00.deviantart.net/d51e/th/pre/i/2018/069/5/b/european_map__arabic__by_annabranco-dc5gve5.jpg"
                            rel="noopener noreferrer"
                            target="_Blank"
                          >
                            <Sample
                              image="https://pre00.deviantart.net/d51e/th/pre/i/2018/069/5/b/european_map__arabic__by_annabranco-dc5gve5.jpg"
                            >
                              <div
                                className="jDHvxb"
                              />
                            </Sample>
                          </a>
                        </div>
                      </Samples Wrapper>
                    </div>
                  </Design Area>
                  <Close Button
                    aria-label="Close"
                    hidden={true}
                    onClick={[Function]}
                    type="button"
                  >
                    <button
                      aria-label="Close"
                      className="hFFHfR"
                      hidden={true}
                      onClick={[Function]}
                      type="button"
                    >
                      Close
                    </button>
                  </Close Button>
                </div>
              </Other Skills Wrapper>
            </div>
          </Mac Screen>
          <Mac Keyboard
            hidden={false}
          >
            <div
              className="goXKNs"
              hidden={false}
            />
          </Mac Keyboard>
          <Keyboard Keys Wrapper
            hidden={false}
          >
            <div
              className="kLquFw"
              hidden={false}
            >
              <Keyboard Keys>
                <div
                  className="eIbhej"
                >
                   . . . . . . . ...
                </div>
              </Keyboard Keys>
              <Keyboard Keys>
                <div
                  className="eIbhej"
                >
                  ... . . . . . . ..
                </div>
              </Keyboard Keys>
              <Keyboard Keys>
                <div
                  className="eIbhej"
                >
                  . . . . . . . . . .
                </div>
              </Keyboard Keys>
              <Keyboard Keys>
                <div
                  className="eIbhej"
                >
                  . . . . . . . . . .
                </div>
              </Keyboard Keys>
              <Keyboard Keys>
                <div
                  className="eIbhej"
                >
                  ------......
                </div>
              </Keyboard Keys>
            </div>
          </Keyboard Keys Wrapper>
        </section>
      </OTHER SKILLS Section>
    </OtherSkills>
  </Provider>
</MockProvider>
`;
